docs/content/getting-started/project-file-reference.mdx

```diff
@@ -0,0 +1,438 @@
+  all configuration files should be placed in the root of your Dagster project
+  directory.
+</Note>
+
+Depending on your use case or if you're using Dagster Cloud, you may also need to add additional configuration files to your project:
+
+<table
+  className="table"
+  style={{
+    width: "100%",
+  }}
+>
+  <thead>
+    <tr>
+      <th
+        style={{
+          width: "20%",
+        }}
+      >
+        File/Directory
+      </th>
+      <th>Description</th>
+      <th
+        style={{
+          width: "10%",
+        }}
+      >
+        OSS
+      </th>
+      <th
+        style={{
+          width: "10%",
+        }}
+      >
+        Cloud
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>dagster.yaml</td>
+      <td>
+        Configures your Dagster instance, including defining storage locations,
+        run launchers, sensors, and schedules. Refer to the{" "}
+        <a href="somelinkhere">dagster.yaml reference</a> for more info,
+        including a list of use cases and available options.
+        <br />
+        <br />
+        For Dagster Cloud <a href="/dagster-cloud/deployment/hybrid">
+          Hybrid deployments
+        </a>, this file can be used to <a href="/dagster-cloud/deployment/agents/customizing-configuration">
+          customize the Hybrid agent
+        </a>.
+      </td>
+      <td>Optional</td>
+      <td>Optional</td>
+    </tr>
+    <tr>
+      <td>dagster_cloud.yaml</td>
+      <td>
+        Defines multiple code locations for Dagster Cloud. For Hybrid
+        deployments, this file can be used{" "}
+        <a href="/dagster-cloud/managing-deployments/setting-environment-variables-dagster-cloud-agents">
+          to manage environment variables/secrets
+        </a>
+        .
+      </td>
+      <td>n/a</td>
+      <td>Optional</td>
+    </tr>
+    <tr>
+      <td>deployment_settings.yaml</td>
+      <td>
+        Configures settings for full deployments in Dagster Cloud, including run
+        queue priority and concurrency limits. Refer to the{" "}
+        <a href="/dagster-cloud/managing-deployments/deployment-settings-reference">
+          Deployment settings reference
+        </a>{" "}
+        for more info.
+        <br />
+        <br />
+        <strong>Note</strong>: This file can be named anything, but we recommend
+        choosing an easily understandable name.
+      </td>
+      <td>n/a</td>
+      <td>Optional</td>
+    </tr>
+    <tr>
+      <td>workspace.yaml</td>
+      <td>
+        Defines multiple code locations for local development or deploying to
+        your infrastructure. Refer to the{" "}
+        <a href="/concepts/code-locations/workspace-files">
+          Workspace file documentation
+        </a>{" "}
+        for more info and available options.
+      </td>
+      <td>Optional</td>
+      <td>n/a</td>
+    </tr>
+  </tbody>
+</table>
+
+---
+
+## Example project structures
```

Comment:
Reviewers - I'm not sure about keeping this section **in this guide**, but I do think it's useful info to have somewhere. Would love your input!

Reply:
Keep it!